common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -11,6 +11,12 @@
     ],
     "sqlState" : "42845"
   },
+  "AGGREGATE_OUT_OF_MEMORY" : {
+    "message" : [
+      "No enough memory for aggregation"
+    ],
+    "sqlState" : "53200"
```

Comment:
Could you assign `sqlState` code from another error class:
https://github.com/apache/spark/blob/26545d000bc880f0d11511d47b37e91ccb5717b7/common/utils/src/main/resources/error/error-classes.json#L80
For instance, "82001" and so on.

Reply:
thanks for pointing it out, updated

Reply:
Okay, will do that independently